app/assets/javascripts/modules/SearchFilter.js

```diff
@@ -0,0 +1,31 @@
+define(['jquery'], function($) {
+
+  'use strict';
+
+  var $element = $('[data-search-filter]'),
+      $trigger = $('[data-search-filter-trigger]'),
+      $triggerIcon = $('[data-search-filter-icon]'),
+      $triggerHeading = $('[data-search-heading-second]'),
+      $target = $('[data-search-filter-target]');
+
+  $element.each(function() {
+    var $this = $(this);
```

Comment:
There are arguments available on the [`each` function in jQuery](http://api.jquery.com/jquery.each/), that will give you the instance of "this" already. So you can remove the `var $this = $(this)` line and just put this in:

```js
$element.each(function(indexInArray, objectInArray) {
  // now anywhere in here you can just use $(objectInArray) instead of $(this)
});
```